cmake/gulark.cmake

```diff
@@ -38,7 +38,7 @@ else()
     FetchContent_Declare(
         ${CONTENT_NAME}
         GIT_REPOSITORY https://github.com/gulrak/filesystem.git
-        GIT_TAG        3d3c02ce35dcc68b5ebb34f21cb1fc507be9a66e
+        GIT_TAG        4e21ab305794f5309a1454b4ae82ab9a0f5e0d25
```

Comment:
Switching to v 1.5.6 which fixed the bug in ghc::filesystem::canonical

https://github.com/gulrak/filesystem/issues/124